### ISSUE TITLE
swagger module miss port

### ISSUE THREAD

**Issue by Debenson** [stoicflame/enunciate#254]
for method SwaggerDeploymentModule#writeTo:
```
model.put("host", getHost());
model.put("schemes", getSchemes());
model.put("basePath", getBasePath());

protected String getHost() {
    String host = this.config.getString("[@host]", null);

    if (host == null) {
      String root = enunciate.getConfiguration().getApplicationRoot();
      if (root != null) {
        try {
          URI uri = URI.create(root);
          host = uri.getHost() ;          
        }
        catch (IllegalArgumentException e) {
          host = null;
        }
      }
    }

    return host;
  }
```

getHost() may miss port part.  
see http://swagger.io/specification/ "host" definition:

The host (name or ip) serving the API. This MUST be the host only and does not include the scheme nor sub-paths. **It MAY include a port**. If the host is not included, the host serving the documentation is to be used (including the port). The host does not support path templating.

**Comment by stoicflame** [comment]
Thanks for the report. Assigning to 2.3.

The workaround is to explicitly provide a host in the configuration:

```xml
<enunciate>
  <modules>
    <swagger host="localhost:8080"/>
```

**Comment by Debenson** [comment]
Thanks.

**Comment by stoicflame** [comment]
Fixed at 17404b9.

**Comment by stoicflame** [comment]
[Enunciate 2.3 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.3.0).
